### PR TITLE
Add author validation task excluded property #61

### PR DESCRIPTION
**Pull request by baumannsven**

### CODE REVIEW

File: tasks/author-validation/documentation.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ Utilized properties
 -------------------
 
 * `phpcq.bin.author-validation` the path to the check-author executable (default: ${phpcq.bin.dir}/check-author.php).
+* `author-validation`.excluded` directories to be excluded from analysis (space separated list, default: empty).
```
Comment (baumannsven) [inline tasks/author-validation/documentation.md:11]: Todo formatting is false

**Comment by discordier** [comment]
Makes sense, however, @dmolineus and I are fully occupied (and only working on v2.0 atm), therefore you should take care about this.
Feel free to release a new bugfix releasse.

**Review by discordier — APPROVED** [review]
Looks good, aside from the formatting issue.

File: tasks/author-validation/documentation.md (hunk 1 above)
Comment (baumannsven) [reply tasks/author-validation/documentation.md]: DONE

**Pull request closed without merging**